src/components/WorkshopPage/WorkshopPage.js

```diff
@@ -214,7 +214,7 @@ function WorkshopPage() {
 			>
 				{item.elements.map(element => <Workshop key={element.title} {...element} />)}
 			</Grid>
-		</React.Fragment>);
+		</React.Fragment>); */
 
 	return (
 		<Container maxWidth='md' style={{ marginBottom: theme.spacing(8) }}>
```

Comment:
Use this as an example of `theme.spacing()` being used. I believe that the default is **8px**, we may have changed it on our own somewhere.